src/server/tran_server.cpp

```diff
@@ -252,8 +252,6 @@ tran_server::get_boot_info_from_page_server ()
   DKNVOLS nvols_perm;
   std::memcpy (&nvols_perm, response_message.c_str (), sizeof (nvols_perm));
 
-  disk_set_page_server_perm_volume_count (nvols_perm);
```

Comment:
Even without this, is the caller, `tran_server::get_boot_info_from_page_server()` still needed?

tran_to_page_request::GET_BOOT_INFO message and communication also don't seem to be needed.

Reply:
Oh, I just see the previous resolved conversation. Please make the issue and ignore this.